custom-hooks/useFetch.js

```diff
@@ -0,0 +1,57 @@
+import { useState, useEffect } from 'react';
```

Comment:
a best use case for `useReducer`

Reply:
Yes, `useReducer` is definitely the best option, but I didn't want to make the code complex for beginners to read. Already hooks are a little hard to understand and using `useReducer` will make the code less beginner-friendly.

Reply:
If you handle multiple states, useReducers are cleaner. The code is easier to reason about.

Reply:
Valid argument @shihabus. Adding useReducer instead of multiple state changes.

Reply:
@invalidtoken For now, let's submit what you've done. Make an enhancement-tagged issue and point to these points please. We can pick it up in future.

Reply:
Added it: https://github.com/Real-Dev-Squad/website-members/issues/20